runtime/interpreter/decode.go

```diff
@@ -227,11 +227,20 @@ func (d *Decoder) decodeValue(v interface{}, path []string) (Value, error) {
 			return d.decodeType(v.Content)
 
 		default:
-			return nil, fmt.Errorf("unsupported decoded tag: %d, %v", v.Number, v.Content)
+			return nil, fmt.Errorf(
+				"unsupported decoded tag (@ %s): %d, %v",
+				strings.Join(path, "."),
```

Comment:
I've made the error messages more descriptive by including the object graph path, which helps with debugging.